jdaviz/configs/cubeviz/helper.py

```diff
@@ -38,6 +38,22 @@ def _set_spectrum_x_axis(self, msg):
             else:
                 viewer.state.x_att_pixel = ref_data.id["Pixel Axis 2 [x]"]
 
+    def load_data(self, data):
+        """
+        Load and parse a data cube with Cubeviz.
+        (Note that only one cube may be loaded per Cubeviz instance.)
+
+        Parameters
+        ----------
+        data : str or `~astropy.io.fits.HDUList`
+            A string file path or astropy FITS object pointing to the
+            data cube.
+        """
+        if len(self.app.state.data_items) != 0:
+            raise RuntimeError('only one cube may be loaded per Cubeviz instance')
```

Comment:
just realized that some day we might want to allow loading 1d spectra and only forbid cubes, but that will require quite a bit more advanced logic, especially on the UI side, so I think this is fine for now until/unless we get a request to re-enable that.